OneCommunityGlobal/HighestGoodNetworkApp
# Shantanu_add_badge_summary

# Description
![bug_details](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/85691749/b3e6989d-8c84-486f-a9df-f859a0022d21)

## Related PRS (if any):
None

## Main changes explained:
Added a preview button with a preview component to view only badges

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as admin user
4. go to dashboard→ Reports→ Reports→Select people 
6. you can see the preview button next to show badges that will show badges of the user

## Screenshots or videos of changes:
![1](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/85691749/ef3e47ad-25fb-4741-8595-4a9391562fdc)
![2](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/85691749/bca43971-46b0-4ce8-8623-4c94f849e281)


## Note:
Sorry I messed up the branch name.
